test/intercept/https/https.request.test.ts

```diff
@@ -1,15 +1,31 @@
 /**
  * @jest-environment node
  */
+import { RequestHandler } from 'express'
 import { RequestInterceptor } from '../../../src'
 import withDefaultInterceptors from '../../../src/presets/default'
 import { InterceptedRequest } from '../../../src/glossary'
 import { prepare, httpsRequest } from '../../helpers'
+import { ServerAPI, createServer, httpsAgent } from '../../utils/createServer'
 
 let requestInterceptor: RequestInterceptor
 let pool: InterceptedRequest[] = []
+let server: ServerAPI
+
+beforeAll(async () => {
+  server = await createServer((app) => {
```

Comment:
it's possible to `createServer` in a general `beforeAll` valid for all tests?
Then in very test where you need the server you will add only the routes

Reply:
I thought about that as well, but decided to spawn a server in each test explicitly. First of all in order to control when a server is needed, as not all tests may require a local server. Secondly, it's always good to start with a non-harmful repetition and either wait until it becomes harmful (i.e. degrades performance), or leave it forever.

Reply:
It can make sense 👍